compose/kubernetes/superset-staging.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: superset-staging
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: superset-staging
+    spec:
+      containers:
+      - name: cloudsql-proxy
+        image: b.gcr.io/cloudsql-docker/gce-proxy:1.09
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+        command: ["/cloud_sql_proxy", "--dir=/cloudsql",
+                  "-instances=soru-176122:us-west1:soru-test=tcp:5432",
+                  "-credential_file=/secrets/cloudsql/credentials.json"]
+        volumeMounts:
+          - name: cloudsql-oauth-credentials
+            mountPath: /secrets/cloudsql
+            readOnly: true
+          - name: ssl-certs
+            mountPath: /etc/ssl/certs
+          - name: cloudsql
+            mountPath: /cloudsql
+      - name: superset
+        image: us.gcr.io/soru-176122/superset-staging:latest
+        imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 8088
+          initialDelaySeconds: 10
+        resources:
+          requests:
+            cpu: 500m
+            memory: 1000Mi
+        ports:
+          - containerPort: 8088
+        env:
+          - name: DB_USER
+            valueFrom:
+              secretKeyRef:
+                name: cloudsql
+                key: username
+          - name: DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: cloudsql
+                key: password
+          - name: SUPERSET_METADATA_DATABASE_URL
+            value: postgres://$(DB_USER):$(DB_PASSWORD)@127.0.0.1:5432/postgres
+          - name: SORU_DB_USER
+            valueFrom:
+              secretKeyRef:
+                name: cloudsql
+                key: username
+          - name: SORU_DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: cloudsql
+                key: password
+          - name: SORU_DATABASE_URL
+            value: postgres://$(SORU_DB_USER):$(SORU_DB_PASSWORD)@127.0.0.1:5432/postgres
+          - name: MAPBOX_API_KEY
+            value: pk.eyJ1IjoiZXJ0YW4iLCJhIjoiY2poOW80eWpwMGYxaDMwbjA5ZW5pNjYycyJ9.JWBiRmCgfR-2EfZEsxminw
+          - name: REDIS_URL
+            value: redis://redis:6379
+          - name: SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                name: ingestion-secret-key-staging
+                key: key
+          - name: SORU_EXTERNAL_HOST
+            value: https://staging.soru.ai
+          - name: SORU_INTERNAL_HOST
+            value: http://django
+      - name: chrome
+        image: us.gcr.io/soru-176122/chrome-staging:latest
+        ports:
+          - containerPort: 9222
+        securityContext:
+          capabilities:
+            add:
+              - SYS_ADMIN
+      - name: screenshotter
+        image: us.gcr.io/soru-176122/screenshotter-staging:latest
```

Comment:
same as above. not using a hash will make debugging really hard in a CD situation